pkgs/servers/matrix-synapse/matrix-appservice-irc/REVISION

```diff
@@ -0,0 +1 @@
+0.30.0
```

Comment:
Nix can return this value from the derivation eg ``nix-instantiate --eval -E 'with (import ./. { }).pkgs; python3Packages.pyopenssl.version'``